cucumber-scala/src/test/scala-2/io/cucumber/scala/ScalaDslStepsTest.scala

```diff
@@ -1,8 +1,8 @@
 package io.cucumber.scala
 
 import io.cucumber.core.backend._
-import org.junit.Assert.{assertEquals, assertTrue}
-import org.junit.Test
+import org.junit.jupiter.api.Assertions.{assertEquals, assertTrue}
```

Comment:
To wit, `assertEquals` has the expected and actual values flipped between JUnit 4 and JUnit 5.